package.json

```diff
@@ -3,20 +3,20 @@
   "version": "0.0.0",
   "private": true,
   "dependencies": {
-    "@angular/animations": "^8.0.0 || ^9.0.0-0",
-    "@angular/cdk": "9.2.1",
-    "@angular/common": "^9.1.2",
-    "@angular/compiler": "^9.1.2",
-    "@angular/core": "^9.1.2",
-    "@angular/forms": "^9.1.2",
-    "@angular/material": "^9.2.1",
-    "@angular/platform-browser": "^9.1.2",
-    "@angular/platform-browser-dynamic": "^9.1.2",
-    "@angular/router": "^9.1.2",
+    "@angular/animations": "^10.0.3",
+    "@angular/cdk": "10.0.1",
+    "@angular/common": "^10.0.3",
+    "@angular/compiler": "^10.0.3",
+    "@angular/core": "^10.0.3",
+    "@angular/forms": "^10.0.3",
+    "@angular/material": "^10.0.1",
+    "@angular/platform-browser": "^10.0.3",
+    "@angular/platform-browser-dynamic": "^10.0.3",
+    "@angular/router": "^10.0.3",
     "core-js": "2",
-    "ngx-pagination-data-source": "^1.1.0",
-    "rxjs": "^6.5.2",
-    "tslib": "^1.10.0",
+    "ngx-pagination-data-source": "file:../ngx-pagination-data-source",
```

Comment:
Requires https://github.com/nilsmehlhorn/ngx-pagination-data-source/pull/2 to be released

Reply:
It's merged and released in v2.0.0

Reply:
I used it as dependency and I also noticed that I forgot to add the new MatNativeDateModule for the datepicker to use.